src/scripts/pagerduty.coffee

```diff
@@ -610,6 +610,14 @@ module.exports = (robot) ->
   robot.respond /who(?:’s|'s|s| is|se)? ((next (on call|oncall|on-call))|((on call|oncall|on-call) next))/i, (msg) ->
     getCustomOncalls 'next', msg
 
+  # who is on call after current shift?
+  robot.respond /who(?:’s|'s|s| is|se)? ((after (on call|oncall|on-call))|((on call|oncall|on-call) after))/i, (msg) ->
+    getCustomOncalls 'after', msg
+
+  # who is on call after current shift?
```

Comment:
- nitpick: wrong comment
- wouldn't be better to use `who was on call before` ? as the past tense `was` is used already for the previous shift?

Reply:
now allowed both, `'s` or `was` or whichever. And fixed comment